android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsViewModel.kt

```diff
@@ -53,7 +53,9 @@ class SurveyQuestionsViewModel(
         }
     }
 
-    fun answerQuestion(questionAnswer: Pair<Int, List<Int>>) {
-        Timber.d("question ${questionAnswer.first} - answers ${questionAnswer.second}")
+    fun answerQuestion(questionAndAnswer: QuestionAndAnswers) {
```

Comment:
mismatching param name 😂 

```suggestion
    fun answerQuestion(questionAndAnswers: QuestionAndAnswers) {
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/119/commits/7807069f58e6f82988f546500774719b119e6fae